web/cypress.json

```diff
@@ -1,10 +1,10 @@
 {
   "env": {
-    "baseUrl": "http://localhost:8080/taskana",
```

Comment:
The baseUrl is the url that is called when the login is executed. The appUrl is used for navigation after a successful login.  
So this is not the REST API url. Since we only interact with the frontend in Cypress, there is no direct communication with the REST API url.

If needed, I can rename the baseUrl to loginUrl.